tests/antithesis/README.md

```diff
@@ -0,0 +1,78 @@
+# Go-based etcd Client Entrypoint
+
+This directory enables integration of Antithesis with etcd. There are 4 containers running in this system: 3 that make up an etcd cluster (etcd0, etcd1, etcd2) and one that "[makes the system go](https://antithesis.com/docs/getting_started/basic_test_hookup/)" (client).
+
+## Quickstart
+
+### 1. Build and Tag the Docker Image
+
+Run this command from the `antithesis/test-template` directory:
+
+```bash
+docker build . -f Dockerfile.client -t etcd-client:latest
+```
+
+### 2. (Optional) Check the Image Locally
+
+You can verify your new image is built:
+
+```bash
+docker images | grep etcd-client
+```
+
+It should show something like:
+
+```
+etcd-client        latest    <IMAGE_ID>    <DATE>
+```
+
+### 3. Use in Docker Compose
+
+If you have a `docker-compose.yml` that defines your etcd cluster plus this client, make sure the `client` service references `etcd-client:latest`, for example:
+
+```yaml
+services:
+  etcd0:
+    image: docker.io/bitnami/etcd:3.5
+    environment:  # ...
+  etcd1:
+    image: quay.io/coreos/etcd:v3.5.0
+    environment:  # ...
+  etcd2:
+    image: docker.io/bitnami/etcd:3.5
+    environment:  # ...
+  client:
+    image: etcd-client:latest
+    container_name: client
+    # no custom entrypoint needed, the Dockerfile sets it
+```
+
+Then run the following command from the root directory for Antithesis tests (`tests/antithesis`):
+
+```bash
+docker-compose up
+```
+
+The client will continuously check the health of the etcd nodes and print logs similar to:
+
+```
+[+] Running 4/4
+ ✔ Container etcd2   Created                                                                                                                                 0.0s 
+ ✔ Container etcd0   Created                                                                                                                                 0.0s 
+ ✔ Container etcd1   Created                                                                                                                                 0.0s 
+ ✔ Container client  Created                                                                                                                                 0.0s 
+Attaching to client, etcd0, etcd1, etcd2
+client  | exec /entrypoint.py: exec format error
+etcd2   | etcd 14:02:27.85 INFO  ==> 
+etcd2   | etcd 14:02:27.86 INFO  ==> Welcome to the Bitnami etcd container
```

Comment:
We don't use bitnami any more, please test the scripts and provide updated results.